### PR TITLE
Pass over Chapter 1: Vectors

### PR DESCRIPTION
**Pull request by shiffman**
Here is my pass over chapter 1 vectors. Some new issues were raised during this pass: nature-of-code/noc-book-2#61 nature-of-code/noc-book-2#60 nature-of-code/book-website-2nd-edition#144 nature-of-code/noc-book-2#58. I'll note a second pass in nature-of-code/noc-book-2#53.

@joeyklee I'll leave this unmerged for now in case it's helpful for you to review my work as a guide for future updates you make.

### CODE REVIEW

File: content/01_vectors.html [hunk 1]
```diff
@@ -345,25 +340,34 @@ <h2>1.2 Vectors for p5.js Programmers</h2>
     <p>becomes …</p>
 
     <pre data-code-language="javascript" data-type="programlisting" class="codesplit">
-let position = createVector(100,100);
-let velocity = createVector(1,3.3);</pre>
+let position = createVector(100, 100);
+let velocity = createVector(1, 3.3);</pre>
+
+    <p>
+      I'll should note that in the above code the vector objects are not created, as you might
```
Comment (joeyklee) [inline content/01_vectors.html]: `I'll should note` ==> "I should note"

File: content/01_vectors.html [hunk 2]
```diff
@@ -475,16 +479,16 @@ <h2>1.3 Vector Addition</h2>
...
-      <code>PVector</code> object as its argument.
+      <code>Vector</code> object as its argument.
     </p>
 
     <pre data-code-language="javascript" data-type="programlisting" class="codesplit">
 class Vector {
 
-  constructor(x_, y_) {
+  constructor(x, y) {
     this.x = x_;
```
Comment (joeyklee) [inline content/01_vectors.html]:
```
this.x = x;
this.y= y;
```

File: content/01_vectors.html [hunk 3]
```diff
@@ -949,17 +949,17 @@ <h5>Example 1.4: Multiplying a vector</h5>
         data-type="indexterm"></a>
 
       <p>
-        Division works just like multiplication—we simply replace the
+        Division works just like multiplication—simply replace the
```
Comment (joeyklee) [inline content/01_vectors.html:952]: Are we removing the references to `simply`?

File: content/01_vectors.html [hunk 4]
```diff
@@ -998,8 +994,8 @@ <h2>1.5 Vector Magnitude</h2>
     <a data-primary="vectors" data-secondary="magnitude" data-type="indexterm"></a>
 
     <p>
-      Multiplication and division, as we just saw, are means by which the length
-      of the vector can be changed without affecting direction. Perhaps you’re
+      Multiplication and division, as I just described, chnage the length
```
Comment (joeyklee) [inline content/01_vectors.html]: `chnage` ==> `change`

File: content/01_vectors.html [hunk 5]
```diff
@@ -1573,14 +1558,14 @@ <h2>1.8 Vector Motion: Acceleration</h2>
...
-      Let’s write every example in the rest of this book without ever touching
+      As an exercise, from this point forward, I’m going to make a rule for myself.
+      I will write every example in the rest of this book without ever touching
       the value of velocity and position (except to initialize them). In other
-      words, our goal now for programming motion is: Come up with an algorithm
-      for how we calculate acceleration and let the trickle-down effect work its
-      magic. (In truth, you’ll find reasons to break this rule, but it’s
-      important to illustrate the principles behind our motion algorithm.) And
-      so we need to come up with some ways to calculate acceleration:
+      words, the goal for programming motion is: Come up with an algorithm
```
Comment (joeyklee) [inline content/01_vectors.html:1564]: I just think this is so cool ❤️

File: content/01_vectors.html [hunk 6]
```diff
@@ -2065,66 +2039,73 @@ <h2>1.10 Interactivity with Acceleration</h2>
...
-      anything equals anything.
+      In order to set the magnitude (whatever it may be) of the acceleration
+      vector, I must first <em>_</em> that direction vector.
+      That’s right, you said it. <em>Normalize</em>. If I can shrink the vector
+      down to its unit vector (of length one) then I can easily scale it to any value. One
+      multiplied by anything equals anything.
     </p>
 
     <pre data-code-language="javascript" data-type="programlisting" class="codesplit">
 let anything = ?????
```
Comment (joeyklee) [inline content/01_vectors.html]: In this case, do we want to stick with the `????` or should we add a comment for a number?

**Comment by joeyklee** [comment]
@shiffman - Just fyi, I will address the comments above before merging. Thanks!

File: content/01_vectors.html (hunk 1 above)
Comment (shiffman) [inline content/01_vectors.html]:
```suggestion
      I'll note that in the above code the vector objects are not created, as you might
```

Comment (shiffman) [reply content/01_vectors.html]: @joeyklee I'll make changes based on your comments but you should feel free to use the "suggest" feature if you like? ⬇

File: content/01_vectors.html (hunk 2 above)
Comment (shiffman) [reply content/01_vectors.html]: Fixed!

File: content/01_vectors.html (hunk 6 above)
Comment (shiffman) [reply content/01_vectors.html]: A convention I do use in the book is "_______" so I'll try that for now:

```javascript
// any number
let anything = _________;
```

File: content/01_vectors.html (hunk 3 above)
Comment (shiffman) [reply content/01_vectors.html:952]: In general yes, but it's a case by case basis and I think it fits in this context?

**Comment by shiffman** [comment]
Thank you for the comments @joeyklee! I made some changes based on them and replied in-line. Feel free to merge whenever!

File: content/01_vectors.html (hunk 3 above)
Comment (joeyklee) [reply content/01_vectors.html:952]: Yep! All good!

**Comment by joeyklee** [comment]
@shiffman - super! Thanks. Merging now!

**Pull request merged**